addons/event/views/event_views.xml

```diff
@@ -692,7 +692,6 @@
                     <filter string="Confirmed" name="confirmed" domain="[('state', '=', 'open')]"/>
                     <filter string="Attended" name="attended" domain="[('state', '=', 'done')]"/>
                     <separator/>
-                    <filter string="Registration Date" name="filter_date_open" date="date_open"/>
```

Comment:
Shouldn't we replace it with a filter on "create_date" ?

Reply:
yes, we can replace it with create_date.